### DISCUSSION TITLE
If you see this please comment

### DISCUSSION THREAD

**Discussion by WildWeazel** [C7-Game/Prototype#184] [General]
**Update: I recommend subscribing to notifications for Discussions under the "Watch" drop-down menu on this repo.**

Just checking to see how visible Discussions are. Not sure if you get a notification for new ones.

After some feedback from multiple people I'm strongly considering moving technical discussion over here, and leave the forum for more user-oriented talk. We could have a meta repo just for issue tracking, discussions, and wiki. This makes is easier to keep track and reference things, and isolates a lot of the eye-glazing stuff for other people to have to sift through.

**Comment by WildWeazel** [comment]
It's also nice that you can label and "upvote" threads here, which would be great for feature/change requests.

**Comment by pcen** [comment]
Didn't get any kind of notification I just happened to see this. I've never used GitHub discussions before but I think it would be nice to consolidate technical discussions into one place, especially given how nicely it integrates with issues.

**Comment by QuintillusCFC** [comment]
Howdy!  I just saw this!  I didn't get any notifications either, but after CFC discussions indicating this area may be opening, clicked on "Discussions" and found it.

**Comment by QuintillusCFC** [comment]
Question: why did my reply immediately get an upvote?  Are you just that fast?

**Reply by WildWeazel** [reply]
Apparently you get your own vote automatically, like on reddit

**Reply by WildWeazel** [reply]
Threaded comments too! Not sure how I feel about that.

**Reply by QuintillusCFC** [reply]
Huh.  I've never actually created a Reddit account (I'm good enough at spending time discussing things at CFC), so I never realized you get a free upvote there.

As a lurker, I don't like threaded comments on Reddit, but that's because you seem to have to dig down one level at a time through the threaded comments and it's just a headache to actually see those sub-threads.  So far this seems to be better because it only goes one level deep, and doesn't hide replies like Reddit.  Although another factor will be whether it's easy to hide excessively long threaded comments.  On Hacker News, it's easier to see the threaded replies than on Reddit... but it's not rare for one top-level comment to spawn a whole tree of sub-comments, and being able to collapse down to just top level ones would be nice.  In those cases, essentially whoever wrote the first comment that spawned a tree dominates the entire comments section, most people never even scroll down to the second top-level comment once a tree reaches a large enough size.

**Comment by WildWeazel** [comment]
~~Do you guys "watch" the repo and/or get notifications of new issues and PRs? I see one reference to new discussions in the notification docs but it doesn't specifically say how to subscribe to them.~~

Ah, I found it! Under the "watch" drop-down there's a custom option where you can check Discussions.

**Comment by maxpetul** [comment]
I just saw this now because pcen mentioned Discussions in the comments for his unit go-to PR. So for visibility I'd rate it as "not very".

**Comment by WildWeazel** [comment]
Just hijacking this post to let @Kright @lucienmaloney @steviek @maxpetul and @Ulrich-Schmidt know that we're still here and active again if you'd like to rejoin us :)

**Comment by FranklinFFR** [comment]
This is my "Hello World" comment.
